starknetup.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/sh
+# shellcheck shell=dash
+
+set -eu
+
+ASDF_INSTALL_DOCS="https://asdf-vm.com/guide/getting-started.html"
+SCARB_UNINSTALL_INSTRUCTIONS="For uninstallation instructions, refer to https://docs.swmansion.com/scarb/download#uninstall"
+# TODO(#2): Link snfoundry uninstall docs once they are available
+STARKNET_FOUNDRY_UNINSTALL_INSTRUCTIONS="Consider removing snforge and sncast binaries from $HOME/.local/bin"
```

Comment:
Maybe something like "try removing"? I don't think there is anything to consider here, we want to nudge people in a righ direction, given there are no instructions on how to do this at the moment